filebeat/module/apache2/access/config/access.yml

```diff
@@ -1,9 +1,9 @@
-- input_type: log
-  paths:
-  {%- for path in paths %}
-   - {{path}}
-  {%- endfor %}
-  exclude_files: [".gz$"]
-  fields:
-    source_type: apache2-access
-    pipeline_id: {{beat.pipeline_id}}
+input_type: log
```

Comment:
Yeah, multiple prospectors didn't really work out with the requirement of being able to override some of the prospector settings when needed.